gode/packages.go

```diff
@@ -134,12 +134,13 @@ func execNpm(args ...string) (string, string, error) {
 func environ() []string {
 	env, path := environPath()
 	env = append(env, "PATH="+prependPathToList(filepath.Dir(nodePath), path))
-	env = append(env, "NPM_CONFIG_ALWAYS_AUTH=false")
-	env = append(env, "NPM_CONFIG_CACHE="+filepath.Join(rootPath, ".npm-cache"))
-	env = append(env, "NPM_CONFIG_REGISTRY="+registry)
-	env = append(env, "NPM_CONFIG_GLOBAL=false")
-	env = append(env, "NPM_CONFIG_ONLOAD_SCRIPT=false")
-	env = append(env, "NPM_CONFIG_AUDIT=false")
+	// TODO (mirande): user lowercase versions - see: https://docs.npmjs.com/misc/config#environment-variables
```

Comment:
This seems like a good permanent change. Remove the TODO?

Reply:
ah, lol. yes will do :+1:

Reply:
✅ done